mcts4py/Solver.py

```diff
@@ -95,6 +95,8 @@ def display_tree_impl(self, depth_limit: int, node: Optional[TNode], indent: str
         if len(children) == 0:
             return
 
+        children.sort(key=lambda c: c.reward/c.n, reverse=True)
```

Comment:
This displays children sorted by the action's average reward. This makes debugging a little easier.

Reply:
👍🏼